src/Video.tsx

```diff
@@ -142,6 +145,35 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
       setRestoreUserInterfaceForPIPStopCompletionHandler,
     ] = useState<boolean | undefined>();
 
+    const _cmcd = useMemo<NativeCmcdConfiguration | undefined>(() => {
```

Comment:
It may be better to merge _cmcd into source to avoid adding this memo. It's maybe better for readability to keep it. I will Approve anyway.

Reply:
9ded21b I have incorporated the PR changes in this commit. Thank you.